packages/core/src/Stores/client-store.js

```diff
@@ -2454,6 +2478,12 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    responseDerivezAvailableAccounts(response) {
+        if (!response.error) {
+            this.derivez_available_accounts = response.trading_platform_accounts;
+        }
+    }
```

Comment:
@shontzu-deriv Please use the `useFetch` hook inside the components to do API calls instead of adding more complexity to stores 🙏🏻 Same for other places as well 🙏🏻 

Let me know if you need help on this 🙇🏻